### PR TITLE
Download results per taks in CSV importable by Edison

### PR DESCRIPTION
**Pull request by geordi**

### CODE REVIEW

File: web/views/teacher.py [hunk 1]
```diff
@@ -298,12 +300,38 @@ def build_score_csv(assignments, filename):
...
         return response
 
+def build_edison_task_score_csv(student_points: Dict, filename: str):
+    """
+    Build CSV file importable by Edison system.
+    Edison requires delimiter set to `;`.
+    """
+    with io.StringIO() as out:
+        for student, points in student_points.items():
+            w = csv.writer(out, delimiter=';')
```
Comment (Kobzol) [inline web/views/teacher.py:310]:
```suggestion
        w = csv.writer(out, delimiter=';')
        for student, points in student_points.items():
```
This should probably be created just once?

**Review by Kobzol — APPROVED** [review]
You will be using this directly by URL for now? :)

File: web/views/teacher.py (hunk 1 above)
Comment (geordi) [reply web/views/teacher.py:310]: Thank you. I didn't catch that. And it surprises me a bit that it works.

**Comment by geordi** [comment]
> 

For now, URL is enough. Will probably add a button to task edit page.

**Review by trnila — APPROVED** [review]

**Pull request merged**